internal/graph/checkingresourcestream.go

```diff
@@ -466,7 +478,10 @@ func (crs *checkingResourceStream) runProcess(alwaysProcess bool) (bool, error)
 		// Set the lookupResult iff the permissionship was a valid permission.
 		var lookupResult *v1.DispatchLookupResourcesResponse
 		if permissionship != v1.ResolvedResource_UNKNOWN {
-			metadata := combineResponseMetadata(rai.reachableResult.Metadata, resultsMetas[rai.reachableResult.Resource.ResourceId])
+			metadata := rai.reachableResult.Metadata
+			metadata = crs.addSkippedDispatchCountToBePublished(metadata)
```

Comment:
do we have concerns about counts being held distorting the next member to return?

Reply:
That was the compromise in order to get accurate counts overall when streaming.